api/cloud-resources/v1beta1/gcpredisinstance_types.go

```diff
@@ -22,6 +22,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// +kubebuilder:validation:Enum=S1;S2;S3;S4;S5;S6;S7;S8;P1;P2;P3;P4;P5;P6;P7
+type GcpRedisTier string
+
+const (
+	GcpRedisTierS1 GcpRedisTier = "S1"
+	GcpRedisTierS2 GcpRedisTier = "S2"
+	GcpRedisTierS3 GcpRedisTier = "S3"
+	GcpRedisTierS4 GcpRedisTier = "S4"
+	GcpRedisTierS5 GcpRedisTier = "S5"
+	GcpRedisTierS6 GcpRedisTier = "S6"
+	GcpRedisTierS7 GcpRedisTier = "S7"
+	GcpRedisTierS8 GcpRedisTier = "S8"
+
+	GcpRedisTierP1 GcpRedisTier = "P1"
+	GcpRedisTierP2 GcpRedisTier = "P2"
+	GcpRedisTierP3 GcpRedisTier = "P3"
+	GcpRedisTierP4 GcpRedisTier = "P4"
+	GcpRedisTierP5 GcpRedisTier = "P5"
+	GcpRedisTierP6 GcpRedisTier = "P6"
+	GcpRedisTierP7 GcpRedisTier = "P7"
+)
+
+func (tier GcpRedisTier) String() string {
```

Comment:
Think it will work the same w/out this method

Reply:
removed